packages/ngtools/webpack/src/paths-plugin.ts

```diff
@@ -109,6 +131,9 @@ export function resolveWithPaths(
     return;
   }
 
+  // TODO: The following is used when there is more than one potential and will not be
+  //       needed once this is turned into a full webpack resolver plugin
+
```

Comment:
Shouldn’t we return the dirname when package.jaon exists?

Reply:
Would you mind opening a followup PR with your suggested changes once this is merged?

Reply:
Sure thing, I can do that 😊